A/ArrayFire/Compat.toml

```diff
@@ -15,7 +15,7 @@ FFTW = "0.2.3-0"
 SpecialFunctions = "0.6-0"
 julia = "0.7-1"
 
-[1]
+["1-1.0.1"]
```

Comment:
Maybe keep these (or at least some) bounds for this new release? Especially since it is a patch release.

Reply:
I'm not sure what Compat.toml is for or what the bounds should be - this PR was generated by a robot :) Do whatever you think is right here.

Reply:
https://github.com/JuliaGPU/ArrayFire.jl/pull/237